src/components/Steppers/Steppers.tsx

```diff
@@ -0,0 +1,198 @@
+  prefix,
+  stepActive,
+  stepFixWidth,
+  stepLive,
+  stepNormal,
+} from './constants'
+
+import './Steppers.scss'
+
+import HuiIcon from '../Icon'
+
+/**
+ * 步骤条
+ * @example
+ *  import Steps from "@/components/Steps";
+
+    const StepsPage: React.FC = () => {
+        return (
+            <Steps
+              items={[
+                  {
+                  title: "步骤名称",
+                  description:'描述文字'
+                  },
+                  {
+                  title: "步骤名称",
+                  description:'描述文字'
+                  },
+                  {
+                  title: "步骤名称",
+                  description:'描述文字'
+                  }
+              ]}
+            />
+        );
+    };
+*/
+const Steppers: React.FC<HuiSteppersProps> = props => {
+  const {
+    onChange,
+    className,
+    items = [],
+    current = 1,
+    disable,
+    showSliceTitle = false,
+  } = props
+  const [cur, setCur] = useState(current)
+  const list = useMemo(() => items.filter(e => e), [items])
+  const overflowAuto = useMemo(() => list.length > 4, [list])
+  const stepsAutoStyle: CSSProperties | undefined = useMemo(
+    () =>
+      (overflowAuto
+        ? {
+          overflowX: 'auto',
+        }
+        : undefined),
+    [overflowAuto],
+  )
+
+  const stepAutoStyle: CSSProperties | undefined = useMemo(
+    () =>
+      (overflowAuto
+        ? {
+          display: 'flex',
+          width: stepFixWidth * list.length,
+        }
+        : undefined),
+    [overflowAuto, list],
+  )
+
+  const onHandleChange = useCallback(
+    (i: number, e: any) => {
+      if (disable) return
+      setCur(i)
+      onChange?.(i, e)
+    },
+    [disable],
+  )
+  const stepItemClassName = useCallback(
+    i => {
+      if (i === cur) return stepLive
+      if (i < cur) return stepActive
+      return stepNormal
+    },
+    [cur],
+  )
+
+  const stepItemIcon = useCallback(
+    i =>
+      (i < cur ? (
+        <HuiIcon
+          name='003-right'
+          size={27}
+          style={{
+            height: 24,
+            width: 24,
+          }}
+        />
+      ) : (
+        i
+      )),
+    [cur],
+  )
+
+  const updateCur = useCallback(() => {
+    if (!isNumber(current)) {
+      warning(`current类型错误： ${current}不是Number类型`)
+      return
+    }
+    const listLength = list?.length
+    if (current > listLength || current < 0) {
+      setCur(current < 0 ? 0 : listLength)
+      warning(`current边界异常： ${current}不在合理区间范围内`)
+      return
+    }
+    setCur(current)
+  }, [current, list])
+
+  const renderItem = (e: StepItem, i: number) => {
+    const {
+      title,
+      description,
+      disable: itemDisable,
+      showSliceTitle: itemShowSliceTitle = false,
+    } = e
+    const index = i + 1
+    const autoStyle: CSSProperties = {
+      width: overflowAuto ? stepFixWidth : 'auto',
+      float: 'left',
+    }
+    const computeClassName = stepItemClassName(index)
+    const computeIconNode = stepItemIcon(index)
```

Comment:
这里感觉computeIconNode像个方法，而stepItemIcon是名词，可以把命名互换下
```suggestion
    const stepItemClassName = computeClassName(index)
    const stepItemIcon = computeIconNode(index)
```

Reply:
更新了